python/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+set(GTSAM_PYTHON_BUILD_DIRECTORY ${PROJECT_BINARY_DIR}/python)
+
+if(GTSAM_BUILD_PYTHON)
```

Comment:
Just an alternative style option, to avoid having to indent the whole file: 

```cmake
if (NOT GTSAM_BUILD_PYTHON)
  return()
endif()
```

Reply:
Done :)